packages/hdwallet-keepkey/src/bitcoin.ts

```diff
@@ -112,6 +83,7 @@ function prepareSignTx(
     } else {
       // BTCSignTxOutputSpend
       newOutput.setScriptType(Types.OutputScriptType.PAYTOADDRESS);
+      assert(output.address != undefined, "Output must have a valid BTC address.");
```

Comment:
preference for strict equality checks. could also just do a falsey `!output.address` check as it is really checking if value exists and not necessarily if valid or not (could update error string to reflect this as well).
```suggestion
      assert(output.address !== undefined, "Output must have a valid BTC address.");
```